doc/lua_api.md

```diff
@@ -564,17 +564,22 @@ Example:
 
     default_cobble.png^[crack:10:1
 
-#### `[combine:<w>x<h>:<x1>,<y1>=<file1>:<x2>,<y2>=<file2>:...`
-
-* `<w>`: width
-* `<h>`: height
-* `<x>`: x position, negative numbers allowed
-* `<y>`: y position, negative numbers allowed
-* `<file>`: texture to combine
+#### `[combine:<w>x<h>:<x1>,<y1>,<w1>=<file1>:<x2>,<y2>,<w2>=<file2>:...`
 
 Creates a texture of size `<w>` times `<h>` and blits the listed files to their
 specified coordinates.
 
+Note: the output texture size may vary if one or more of `<wN>` is provided.
+The aspect ratio is preserved.
+
+* `<xN>`: X offset of insertion, negative numbers allowed
+* `<yN>`: Y offset of insertion, negative numbers allowed
+* `<wN>` (optional): Expected texture width
+   * Scales the output image if `<wN> != input texture width` to perform lossless blit.
+   * Supported since protocol version 46.
```

Comment:
(assuming that's true)
```suggestion
   * Supported since protocol version 46. Old clients will fail when encountering this.
```

Reply:
In fact, the parameter is simply ignored. Hence there's no compat code needed. I'll add a comment to clarify this.

Reply:
Comment added in [3128beb](https://github.com/minetest/minetest/pull/15084/commits/3128beba7faf124e7134b846f200bfba1db5acbd)